packages/create-svelte/templates/skeletonlib/package.template.json

```diff
@@ -15,5 +22,6 @@
 		"typescript": "^4.9.3",
 		"vite": "^4.0.0"
 	},
+	"svelte": "./dist/index.js",
```

Comment:
this should be unnecessary with the `svelte` export condition IIUC?

```suggestion
```

Reply:
backwards compatibility for old bundler versions not supporting it yet